phing/build.yml

```diff
@@ -31,6 +31,8 @@ blt:
     local: ${repo.root}/blt/project.local.yml
     example-local: ${repo.root}/blt/example.project.local.yml
     schema-version: ${repo.root}/blt/.schema_version
+  config-dirs:
```

Comment:
At this point, no, but I figured to future-proof this a bit it may make sense to allow additional keys to be added here in the future. But the `sites` key could certainly simply be placed under the existing `config-files` key.